pydata_sphinx_theme/_templates/switcher.html

```diff
@@ -0,0 +1,53 @@
+<div class="dropdown">
+    <button type="button" class="btn btn-primary btn-sm navbar-btn dropdown-toggle" id="switcher_button" data-toggle="dropdown">
+        {{ switcher_prefix }}{{ release }}
+        <span class="caret"></span>
+    </button>
+    <div id="switcher" class="dropdown-menu list-group-flush py-0" aria-labelledby="switcher_button">
+    <!-- dropdown will be populated by javascript on page load -->
+    </div>
+</div>
+
+<script type="text/javascript">
+// Function to construct the target URL from the JSON components
+function buildURL(template, entry) {
+    //
+    var parts = template.split(/[\{\}]/);
+    version_ix = parts.indexOf("version");
+    language_ix = parts.indexOf("language");
+    if (version_ix >= 0) {
+        parts[version_ix] = entry.version;
+    }
+    if (language_ix >= 0) {
+        parts[language_ix] = entry.language;
+    }
+    return parts.join("");
+}
+// Function to populate the (release, language) version switcher
+(function () {
+    // get JSON config
+    $.getJSON("{{ switcher_json_url }}", function(data, textStatus, jqXHR) {
```

Comment:
Considering that `switcher_json_url = "/_static/switcher.json"` and this code will be hosted at `/language/version/`, shouldn't that JSON url be relative instead?

Reply:
If you always want it to point to the same `versions.json` file (so you only have to update a single file if a new release needs to be added), I think it needs to be a absolute path?

While trying this out, I actually accidentally specified this path in conf.py as a relative path, and then the version dropdown also only works on top-level pages. Because once you are on a page in a subdirectory, the relative path isn't correct.

Reply:
I think I didn't express myself properly.

I was trying to say that the path should be `/en/latest/_static/switcher.json` on one side.

On the other side, I thought that each documentation version would have a `switcher.json` file containing only the versions released until that moment. So, as you already mentioned, it would get outdated once a new version is released. This is what Read the Docs solves by using an API endpoint. Now, I understood what you said regarding using _the same_ JSON file for all the versions --which makes sense to me. However, the path still requires `/en/latest/` prefix. At least to work on Read the Docs 😄